frontend/src/libs/components/header/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+import { useAppDispatch, useCallback } from '~/libs/hooks/hooks.js';
+import { actions as authActions } from '~/slices/auth/auth.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  trigger: {
+    handleToggleModalOpen: () => void;
+    isOpen: boolean;
+  };
+};
+
+const DropDown: React.FC<Properties> = ({ trigger }) => {
```

Comment:
I guess `Dropdown` would be better, because such name is commonly used for that component 🙂